packages/loaders/json-schema/src/getComposerFromJSONSchema.ts

```diff
@@ -680,30 +680,38 @@ export function getComposerFromJSONSchema(
           };
         }
         case 'object': {
-          switch (subSchema.title) {
+          switch (subSchema.title.toLowerCase()) {
```

Comment:
Is this really necessary, because we already sanitize this kind of names in here?
https://github.com/Urigo/graphql-mesh/blob/master/packages/utils/src/sanitize-name-for-graphql.ts#L1

Reply:
schema.title is not sanitized at this stage

we are traversing generated JSON schema here and in case it contains generated root type Query and one of components has name Query we could miss assign component as a root type

Also for some reason root type name for the added open api example is generated with lowercased title